scopes.md

```diff
@@ -0,0 +1,138 @@
+
+diaspora* also supports `profile` and `email` scopes according to [OpenID Connect Core 1.0][connect-scopes].
+
+Other scopes are diaspora* API specific. All scopes besides `openid` are optional.
+
+Each API endpoint requires at least one scope to be approved. If the access token used for making a request wasn't approved for the scope of the respective endpoint then `403` status will be responded.
+
+## Scopes list
+
+### contacts:read
+
+This scope gives access to the following requests:
+- Read aspect list
+- Read aspects membership
+- Read private user profiles of contacts
+- Read contacts of contacts when allowed by contact's aspect setting
+- User search will include contacts who are hidden from public search
+
+### contacts:modify
+
+This scope gives access to the following requests:
+- Add new aspects
+- Rename aspects and change aspects properties
+- Delete aspects
+- Add/remove person to/from aspects
+
+### conversations
+
+This scope gives access to private messaging, i.e. the following requests:
+- Create conversations
+- Send private messages
+
+### email
+
+This scope is defined by the OpenID Connect spec. It allows reading user's own email address using UserInfo route or from ID Token.
+
+### interactions
+
+This scope gives access to the following requests:
+- Create/delete comments
+- Report posts/comments
+- Post/remove likes
+- Subscribe/mute posts
+- Hide posts from streams
+
+### openid
+
+This scope is defined by the OpenID Connect spec. This is a mandatory scope. It gives access to OpenID Connect authentication.
+
+### private
+
+This scope gives access to the following requests:
+- Read private posts
+- Read private posts intercations (likes, comments)
+
+Reading private posts of contacts additionaly requires `contacts:read` scope.
+
+### public
+
+This scope gives access to the following requests:
+- Read public profiles of users
+- Perform search among publicly searchable profiles
+- Read public posts
+- Read reshares
+- Read public posts intercations (likes, comments)
+
+### posts:publish
+
+This scope gives access to the following requests:
+- Create posts
+- Create reshares
+- Post photos
+
+Create private posts additionaly requires `contacts:read` scope.
+
+### posts:modify
+
+This scope gives access to the following requests:
+- Delete posts
+- Delete photos
+
+### profile
+
+This scope is defined by the OpenID Connect spec. It allows reading user claims (user properties). Claims are read using UserInfo route or from ID Token.
+
+User claims defined by the OpenID Connect are somewhat similar to users own profile data. Generally this scope means "Allow reading user's own profile". Besides user claims defined by the standard this scope also gives access to "read own profile" diaspora* API endpoint.
+
+Supported claims by diaspora:
```

Comment:
Given this a document about "stuff I can request accesss to", maybe we should clarify here whether or not a claim must be explicitly requested or is just given through requesting the scope. I actually don't remember myself but I assume the latter, so something like: "This gives read access to the follow claims supported by diaspora*:"

Reply:
This is actually a part of the `profile` scope specification and the first sentence of this section says pretty much this thing:
> This scope is defined by the OpenID Connect spec. It allows reading user claims (user properties). Claims are read using UserInfo route or from the ID Token.

Reply:
Sure, I don't see why being clear about it here would hurt though :)

Reply:
Done